src/test/mir-opt/dest-prop/union.main.DestinationPropagation.diff

```diff
@@ -17,23 +17,29 @@
       }
   
       bb0: {
-          StorageLive(_1);                 // scope 0 at $DIR/union.rs:13:9: 13:11
-          StorageLive(_2);                 // scope 0 at $DIR/union.rs:13:23: 13:28
-          _2 = val() -> bb1;               // scope 0 at $DIR/union.rs:13:23: 13:28
+-         StorageLive(_1);                 // scope 0 at $DIR/union.rs:13:9: 13:11
```

Comment:
interesting that we're now not marking the `_1` local as live anymore, but using it right below.  Maybe that is what would be causing miri to appear to care about unions being not assigned to?